code/__DEFINES/components.dm

```diff
@@ -1,440 +0,0 @@
-//shorthand
```

Comment:
You need to revert this file.

Reply:
i just removed the file from the PR and its saying it compiled fine earlier.